chainer/links/rnn/n_step_rnn.py

```diff
@@ -51,21 +51,26 @@ class NStepRNNBase(link.ChainList):
     """
 
     def __init__(self, n_layers, in_size, out_size, dropout, **kwargs):
-        if kwargs:
-            argument.check_unexpected_kwargs(
-                kwargs,
-                use_cudnn='use_cudnn argument is not supported anymore. '
-                'Use chainer.using_config',
-                use_bi_direction='use_bi_direction is not supported anymore',
-                activation='activation is not supported anymore')
-            argument.assert_kwargs_empty(kwargs)
+        initialW, initial_bias = argument.parse_kwargs(
+            kwargs,
+            ('initialW', None),
+            ('initial_bias', None),
+            use_cudnn='use_cudnn argument is not supported anymore. '
+            'Use chainer.using_config',
+            use_bi_direction='use_bi_direction is not supported anymore',
+            activation='activation is not supported anymore',
+        )
```

Comment:
Optional: The signature could be
```
__init__(self, n_layers, in_size, out_size, dropout, *, initialW=None, initial_bias=None, **kwargs)
```
as Python 2 support has been dropped.

Reply:
9fb69b3 0c87915
Updated to use keyword arguments.